io-engine/src/logger.rs

```diff
@@ -420,26 +428,50 @@ impl LogFormat {
 ///
 /// We might want to suppress certain messages, as some of them are redundant,
 /// in particular, the NOTICE messages as such, they are mapped to debug.
-pub fn init_ex(level: &str, format: LogFormat) {
+pub fn init_ex(level: &str, format: LogFormat, events_url: Option<url::Url>) {
     // Set up a "logger" that simply translates any "log" messages it receives
     // to trace events. This is for our custom spdk log messages, but also
     // for any other third party crates still using the logging facade.
+
     LogTracer::init().expect("failed to initialise LogTracer");
 
     // Create a default subscriber.
-    let builder = tracing_subscriber::fmt::Subscriber::builder()
+    let builder = tracing_subscriber::fmt::layer()
         .with_span_events(FmtSpan::FULL)
-        .event_format(format);
+        .event_format(format)
+        .with_filter(filter_fn(|metadata| {
```

Comment:
Instead of modifying logger, can we consider using existing functionality to selectively rise and lower level for specific target? Probably, changing RUST_LOG defaults to exclude "mbus-events-target" would work.

If changing RUST_LOG defaults proves to be no good for this particular case, could you move logger changes to separate commit?

Reply:
Changing the defaults to exclude "mbus-event-target" is removing the eventing traces from all the layers and the events are not generated.

Added eventing integration changes to separate commit now.